src/fitnesseMain/badCodeToTestCheckstyle.java

```diff
@@ -0,0 +1,56 @@
+package fitnesseMain;
+
+import java.util.*; //BAD: star imports
+
+class  Hello
+{
+  public final static String hello = "helloWorld!"; //BAD: modifier order
+
+  public String badCommaWhiteSpaceBeforeAndAfter(String input ,String anotherInput)
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') ',' is not followed by whitespace. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.cloudmok.org/coding_rules#rule_key=checkstyle%3AWhite_Space_After_Custom)